aio/content/guide/upgrade-lazy.md

```diff
@@ -238,20 +236,20 @@ import { BrowserModule } from '@angular/platform-browser';
   ]
 })
 export class MainAngularModule {
-  // Empty placeholder method to prevent the `Compiler` from complaining.
+  // Empty placeholder method to satisfy the `Compiler`.
   ngDoBootstrap() {}
 }
 </code-example>
 
 This bare minimum `NgModule` imports `BrowserModule`, the module every Angular browser-based app
 must have. It also defines an empty `ngDoBootstrap()` method, to prevent the {@link Compiler
-Compiler} from complaining. This is necessary, because the module will not have a `bootstrap`
+Compiler} from returning errors. This is necessary because the module will not have a `bootstrap`
```

Comment:
You don't like the idea of a coplaining compiler, do you? 😛

Reply:
Ha! I prefer happy compilers.